airbyte-integrations/connectors/source-bigquery/integration_tests/invalid_config.json

```diff
@@ -1,3 +1,8 @@
 {
-  "todo-wrong-field": "this should be an incomplete config file, used in standard tests"
-}
+  "dataset_id": "test.test_airbyte_dataset",
+  "project_id": "test",
+  "credentials_json": "{\"type\": \"service_account\",\"project_id\": \"test\",\"private_key_id\": \"1\",\"private_key\": \"-----BEGIN PRIVATE KEY-----\n-----END PRIVATE KEY-----\n\",\"client_email\": \"test@test.iam.gserviceaccount.com\",\"client_id\": \"1\",\"auth_uri\": \"https://accounts.google.com/o/oauth2/auth\",\"token_uri\": \"https://oauth2.googleapis.com/token\",\"auth_provider_x509_cert_url\": \"https://www.googleapis.com/oauth2/v1/certs\",\"client_x509_cert_url\": \"\",\"universe_domain\": \"googleapis.com\"}",
+  "streams": [],
+  "replication_method": {"method": "History"},
+  "slice_range": "-1"
```

Comment:
Does this is the indicator the source should look at the data -1 day from the starting point? If so, can we have the `lookback window` instead, having the `slice_range` always a positive int value?

Reply:
No this is just to test if check connection fails on an invalid value of `slice_range`